src/EventStore.Core.Tests/Services/Transport/Http/build_requested_url_should.cs

```diff
@@ -52,43 +52,43 @@ public void with_advertise_ip_and_http_port_set_both_host_and_port_is_changed()
         }
 
         [Test]
-        public void with_port_forward_header_only_port_is_changed()
+        public void with_port_forward_header_only_nothing_is_changed()
         {
             var headers = new NameValueCollection { { "X-Forwarded-Port", "4321" } };
             var requestedUri =
                 HttpEntity.BuildRequestedUrl(inputUri, headers, null, 0);
 
-            Assert.AreEqual(new Uri("http://www.example.com:4321/path/?key=value#anchor"), requestedUri);
+            Assert.AreEqual(new Uri("http://www.example.com:1234/path/?key=value#anchor"), requestedUri);
         }
 
         [Test]
         public void non_integer_port_forward_header_is_ignored()
         {
-            var headers = new NameValueCollection { { "X-Forwarded-Port", "abc" } };
+            var headers = new NameValueCollection { {"X-Forwarded-Proto", "http"}, {"X-Forwarded-Host","www.example.com"}, { "X-Forwarded-Port", "abc" }};
             var requestedUri =
                 HttpEntity.BuildRequestedUrl(inputUri, headers, null, 0);
 
             Assert.AreEqual(inputUri, requestedUri);
         }
 
         [Test]
-        public void with_proto_forward_header_only_scheme_is_changed()
+        public void with_proto_forward_header_only_nothing_is_changed()
```

Comment:
See comment above about "nothing_is_changed".
Here the scheme was changed from `http` to `https`.